frontend/actions/index.js

```diff
@@ -31,3 +49,68 @@ export const fetchPointsProducts = () => (dispatch, getState) => {
     });
 };
 
+/**
+ * Fetches Smile Customer from smile
+ * @returns {Function}
+ */
+export const fetchSmileCustomer = () => (dispatch, getState) => {
+  const smileCustomerState = getSmileCustomerState(getState()) || {};
+
+  if (smileCustomerState.isFetching || smileCustomerState.pointsProducts) {
```

Comment:
```suggestion
  if (smileCustomerState.isFetching || smileCustomerState.customer) {
```

Reply:
I removed the check for the customer because the customer may be in redux but will have to be updated. Like after points are earned or points are spent.